### PR TITLE
STAC Common Metadata

### PR DESCRIPTION
**Pull request by m-mohr**
**Related Issue(s):** #659 

**Proposed Changes:**

1. Made centralized list of common metadata fields.
2. To be discussed: intruments and platform are not required any longer.

**PR Checklist:**

- [ ] This PR has **no** breaking changes.
- [x] I have added my changes to the [CHANGELOG](https://github.com/radiantearth/stac-spec/blob/dev/CHANGELOG.md) **or** a CHANGELOG entry is not required.
- [x] API only: I have run `npm run generate-all` to update the [generated OpenAPI files](https://github.com/radiantearth/stac-spec/blob/dev/api-spec/README.md#openapi-definitions).

### CODE REVIEW

File: extensions/eo/README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ A lot of EO data will have common metadata across many Items.
 It is not necessary, but recommended to use the [Commons extension](../commons/README.md)
 (see chapter "Placing common fields in Collections").
 
-If the data has been collected by a satellite, it is strongly recommended to use the [`sat` extension](../sat/README.md), which in turn requires the [`instrument` extension](../instrument/README.md). If the data has been collected on an airborne platform is is strongly recommended to use the [`instrument` extension](../instrument/README.md).
+If the data has been collected by a satellite, it is strongly recommended to use the [`sat` extension](../sat/README.md), which in turn requires the [Instrument Fields](../item-spec/common-metadata.md#instrument). If the data has been collected on an airborne platform is is strongly recommended to use the [Instrument Fields](../item-spec/common-metadata.md#instrument).
```
Comment (scisco) [inline extensions/eo/README.md]: "platform is is" > "platform it is"

File: item-spec/common-metadata.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### Date and Time Range
+
+- [JSON Schema](json-schema/datetimerange.json)
+
+While a STAC item can have a nominal datetime describing the capture, this extension allows an item to have a range
+of capture datetimes. An example of this is the [MODIS 16 day vegetation index product.](https://lpdaac.usgs.gov/products/mod13q1v006/).
+The datetime property in a STAC item and these fields are not mutually exclusive.
+
+**Important:** Using one of the fields REQUIRES to include the other fields as well to enables a user to search STAC records by the provided times.
```
Comment (scisco) [inline item-spec/common-metadata.md]: what does "the other fields" refer to here?

File: item-spec/common-metadata.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+| Field Name     | Type   | Description                                                  |
+| -------------- | ------ | ------------------------------------------------------------ |
+| start_datetime | string | The first or start date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+| end_datetime   | string | The last or end date and time for the item, in UTC. It is formatted as `date-time` according to [RFC 3339, section 5.6](https://tools.ietf.org/html/rfc3339#section-5.6). |
+
+
+## Licensing
+
+Information about the license(s) of the data, which not the necessarily the same license that applies to the metadata.
```
Comment (scisco) [inline item-spec/common-metadata.md]: "which not the" -> "which is not "

File: item-spec/common-metadata.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+#### Relation types
+
+| Type         | Description                                                  |
+| ------------ | ------------------------------------------------------------ |
+| license      | The license URL(s) for the item SHOULD be specified if the `license` field is set to `proprietary` or `various`. If there is no public license URL available, it is RECOMMENDED to supplement the STAC Item with the license text in a separate file and link to this file. |
+
+
+## Provider
+
+Information about the organizations capturing, producing, processing, hosting and publishing this data.
```
Comment (scisco) [inline item-spec/common-metadata.md]: "hosting and publishing" -> "hosting or publishing"

File: item-spec/common-metadata.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+
+- [JSON Schema](json-schema/provider.json)
+
+| Field Name | Type   | Description |
+| ---------- | ------ | ----------- |
+| providers  | [[Provider Object](#provider-object)] | A list of providers, which may include all organizations capturing or processing the data or the hosting provider. Providers should be listed in chronological order with the most recent provider being the last element of the list.  |
+
+#### Provider Object
+
+The object provides information about a provider. A provider is any of the organizations that captured or processed the content of the collection and therefore influenced the data offered by this collection. May also include information about the final storage provider hosting the data.
```
Comment (scisco) [inline item-spec/common-metadata.md]: "A provider is any of the organizations that captured or processed the content of the collection and therefore influenced the data offered by this collection", change it to present tense to match the rest of the document: "A provider is any of the organizations that captures or processes"

File: item-spec/common-metadata.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+`landsat-8` (Landsat-8), `sentinel-2a` and `sentinel-2b` (Sentinel-2), `terra` and `aqua` (part of NASA EOS, 
+carrying the MODIS instruments), `mycorp-uav-034` (hypothetical drone name), and `worldview02` 
+(Maxar/DigitalGlobe WorldView-2).
+
+**instruments** is an array of all the sensors used in the creation of the data. For example, data from the Landsat-8
+platform is collected with the OLI sensor as well as the TIRS sensor, but the data is distributed together so would be
+specified as `['oli', 'tirs']`. Other instrument examples include `msi` (Sentinel-2), `aster` (Terra), and `modis`
+(Terra and Aqua), `c-sar` (Sentinel-1) and `asar` (Envisat).
+
+**constellation** is the name of a logical collection one or more platforms that have similar payloads and have
```
Comment (scisco) [inline item-spec/common-metadata.md]: "a logical collection one or more" -> "a logical collection of one or more"

File: item-spec/common-metadata.md [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+(Maxar/DigitalGlobe WorldView-2).
+
+**instruments** is an array of all the sensors used in the creation of the data. For example, data from the Landsat-8
+platform is collected with the OLI sensor as well as the TIRS sensor, but the data is distributed together so would be
+specified as `['oli', 'tirs']`. Other instrument examples include `msi` (Sentinel-2), `aster` (Terra), and `modis`
+(Terra and Aqua), `c-sar` (Sentinel-1) and `asar` (Envisat).
+
+**constellation** is the name of a logical collection one or more platforms that have similar payloads and have 
+their orbits arranged in a way to increase the temporal resolution of acquisitions of data with similar geometric and 
+radiometric characteristics. This field allows users to search for related data sets without needing to specify which
```
Comment (scisco) [inline item-spec/common-metadata.md]: "needing to" -> "the need to"

File: item-spec/common-metadata.md [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+**constellation** is the name of a logical collection one or more platforms that have similar payloads and have 
+their orbits arranged in a way to increase the temporal resolution of acquisitions of data with similar geometric and 
+radiometric characteristics. This field allows users to search for related data sets without needing to specify which 
+specific platform the data came from, for example, from either of the Sentinel-2 satellites. Examples include `landsat-8` 
+(Landsat-8, a constellation consisting of a single platform), `sentinel-2`
+([Sentinel-2](https://www.esa.int/Our_Activities/Observing_the_Earth/Copernicus/Sentinel-2/Satellite_constellation)), 
+`rapideye` (operated by Planet Labs), and `modis` (NASA EOS satellites Aqua and Terra).  In the case of `modis`, this
+is technically referring to a pair of sensors on two different satellites, whose data is combined into a series of 
+related products. Additionally, the Aqua satellite is technically part of the A-Train constellation and Terra is not 
+part of a constellation, but these combine to form the logical collection referred to as MODIS.
```
Comment (scisco) [inline item-spec/common-metadata.md]: "these combine" -> "these are combined"

File: item-spec/common-metadata.md (hunk 2 above)
Comment (m-mohr) [reply item-spec/common-metadata.md]: If you use start_datetime you need to add end_datetime, too. And vice-versa.

**Comment by m-mohr** [comment]
Thanks @scisco for the review. I fixed all the raised issues.

**Pull request merged**